cvmfs/catalog_mgr_rw.cc

```diff
@@ -12,9 +12,12 @@
 #include <cassert>
 #include <cstdio>
 #include <cstdlib>
+#include <ctime>
```

Comment:
Ah, indeed.  But that's a flaw already in the existing code.  Please make a separate pull request for fixes in which this missing include can be added.